src/app/admin/options.tsx

```diff
@@ -0,0 +1,94 @@
+import { SecurePassword } from "@blitzjs/auth/secure-password";
+
+import { NextAdminOptions } from "@premieroctet/next-admin";
+import PasswordInput from "./components/PasswordInput";
+
+export const options: NextAdminOptions = {
+  title: "⚡️ Admin Panel",
+  model: {
+    User: {
+      aliases: {
+        newPassword: "Password",
+      },
+      edit: {
+        display: [
+          "name",
+          "email",
+          "newPassword",
+          "role",
+          "tokens",
+          "sessions",
+          "exportDestinations",
+          "bundleExports",
+          "createdAt",
+          "updatedAt",
+        ],
+        customFields: {
+          newPassword: {
+            input: <PasswordInput />,
+            required: true,
+          },
+        },
+        hooks: {
+          beforeDb: async (data, mode, request) => {
+            try {
+              console.log("before DB");
+              const newPassword = data.newPassword;
```

Comment:
Is this that thing you said wasn't working? Is it a convention for this admin panel to change password like this?